server/src/main/java/org/elasticsearch/index/engine/LiveVersionMapArchive.java

```diff
@@ -40,13 +40,26 @@ default boolean isUnsafe() {
     }
 
     /**
-     * Returns how much memory is currently being used by the archive and would be freed up after
-     * unpromotables are refreshed.
+     * Returns the total memory usage if the Archive.
      */
     default long getMemoryBytesUsed() {
         return 0L;
     }
 
+    /**
+     * Returns how much memory could be freed up by creating a new commit and issuing a new unpromotable refresh.
+     */
+    default long getReclaimableMemoryBytes() {
```

Comment:
Nit for names: We are using all of these terms for methods and variables:
* Memory
* MemoryBytes
* Bytes
* RamBytes

It would be great if we could settle on a single terminology.

Reply:
I couldn't agree more! In the existing code we have that too. But to reduce noise in the PR, let's for now consider that list as synonyms! I'll make sure to use the same terminology in a follow up PR. That might make reviewing much easier.